client/types/charges.d.ts

```diff
@@ -28,7 +28,7 @@ interface ChargeRefunds {
 
 export interface PaymentMethodDetails {
 	card?: any;
-	type: 'card';
+	type: 'card_present' | 'card';
```

Comment:
What about 'interac_present'?

Reply:
I've found [this](https://github.com/Automattic/woocommerce-payments/blob/develop/client/payment-details/payment-method/index.js#L47) and [this](https://github.com/Automattic/woocommerce-payments/blob/develop/client/payment-details/payment-method/index.js#L24).

So it looks like it's missing more than interac. Should I add all of them? 🤔

Reply:
Add as many as you are aware of =)